config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+
+  root 'users#index'
+
+  get 'users/user/:id', to: 'users#user'
+
+  resources :users do
+    resources :posts
```

Comment:
- I would advise you to run `Rails route`s at the end of the change to confirm this output:
![Capture](https://user-images.githubusercontent.com/63972670/215617835-32a924f4-6b9a-47d7-85e4-25fd08f44c42.JPG)